### PR TITLE
Kotlin 1.5 support

### PR DESCRIPTION
**Pull request by restingbull**
Built on https://github.com/bazelbuild/rules_kotlin/pull/544.

### CODE REVIEW

**Comment by restingbull** [comment]
@Bencodes, sync'd with last pr.

File: src/main/starlark/release_archive/repository.bzl [hunk 1]
```diff
@@ -65,6 +65,12 @@ def archive_repository_implementation(repository_ctx):
             archive = release_artifact,
         )
 
+        # update release when the contents change.
+        return [
+            repository_ctx.path("../../%s" % release_archive.workspace_root),
```
Comment (cgruber) [inline src/main/starlark/release_archive/repository.bzl:70]: Well this seems important. I'm a little shocked it didn't have this before.

**Review by cgruber — APPROVED** [review]
Looks great.

**Comment by cgruber** [comment]
Can @Bencodes and @restingbull here do `@googlebot I consent` in a comment, to satisfy the CLAbot, please?

**Comment by jeffzoch** [comment]
not sure if its a bug or im doing something wrong but when i try this branch out and use language level 1.5 I see: 
```
KotlinCompile //projects/shared-models:shared-models-lib { kt: 1, java: 0, srcjars: 0 } for darwin failed: (Exit 1): build failed: error executing command
  (cd /private/var/tmp/_bazel_j.zoch/8bdf7aad5539a314ee98a49e42674806/execroot/sfdc_bazel && \
  exec env - \
    LC_CTYPE=en_US.UTF-8 \
    REPOSITORY_NAME=io_bazel_rules_kotlin \
  bazel-out/host/bin/external/io_bazel_rules_kotlin/src/main/kotlin/build '--flagfile=bazel-out/darwin-fastbuild/bin/projects/libs/dataprotection/shared-models/shared-models-lib-kt-java.jar-0.params')
Execution platform: @local_config_platform//:host
exception: java.io.EOFException
	at java.base/java.io.ObjectInputStream$BlockDataInputStream.readFully(ObjectInputStream.java:3317)
	at java.base/java.io.ObjectInputStream.readFully(ObjectInputStream.java:1155)
	at org.jetbrains.kotlin.utils.PluginUtilsKt.decodePluginOptions(pluginUtils.kt:42)
	at org.jetbrains.kotlin.kapt3.Kapt3CommandLineProcessor.processOption(Kapt3Plugin.kt:73)
	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParserKt.processCompilerPluginsOptions(PluginCliParser.kt:125)
	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.processPluginOptions(PluginCliParser.kt:83)
	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.loadPlugins(PluginCliParser.kt:72)
	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.loadPluginsSafe(PluginCliParser.kt:44)
	at org.jetbrains.kotlin.cli.common.CLICompiler.loadPlugins(CLICompiler.kt:183)
	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:69)
	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:52)
	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:92)
	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:44)
	at org.jetbrains.kotlin.cli.common.CLITool.exec(CLITool.kt:98)
	at io.bazel.kotlin.compiler.BazelK2JVMCompiler.exec(BazelK2JVMCompiler.kt:30)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.bazel.kotlin.builder.toolchain.KotlinToolchain$KotlinCliToolInvoker.compile(KotlinToolchain.kt:186)
	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1$1$1.invoke(compilation_task.kt:184)
	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1$1$1.invoke(compilation_task.kt:184)
	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.executeCompilerTask(CompilationTaskContext.kt:122)
	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.executeCompilerTask$default(CompilationTaskContext.kt:114)
	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1.invoke(compilation_task.kt:182)
	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1.invoke(compilation_task.kt:166)
	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.execute(CompilationTaskContext.kt:148)
	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.execute(CompilationTaskContext.kt:140)
	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt.runPlugins(compilation_task.kt:166)
	at io.bazel.kotlin.builder.tasks.jvm.KotlinJvmTaskExecutor.execute(KotlinJvmTaskExecutor.kt:54)
	at io.bazel.kotlin.builder.tasks.KotlinBuilder.executeJvmTask(KotlinBuilder.kt:232)
	at io.bazel.kotlin.builder.tasks.KotlinBuilder.build(KotlinBuilder.kt:130)
	at io.bazel.kotlin.builder.tasks.CompileKotlin.invoke(CompileKotlin.kt:27)
	at io.bazel.worker.PersistentWorker$compileWork$2$result$1.invoke(PersistentWorker.kt:96)
	at io.bazel.worker.PersistentWorker$compileWork$2$result$1.invoke(PersistentWorker.kt:94)
	at io.bazel.worker.WorkerContext$TaskContext.resultOf(WorkerContext.kt:128)
	at io.bazel.worker.WorkerContext.doTask(WorkerContext.kt:156)
	at io.bazel.worker.PersistentWorker$compileWork$2.invokeSuspend(PersistentWorker.kt:94)
	at io.bazel.worker.PersistentWorker$compileWork$2.invoke(PersistentWorker.kt)
	at io.bazel.worker.PersistentWorker$compileWork$2.invoke(PersistentWorker.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:165)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at io.bazel.worker.PersistentWorker.compileWork(PersistentWorker.kt:93)
	at io.bazel.worker.PersistentWorker.access$compileWork(PersistentWorker.kt:45)
	at io.bazel.worker.PersistentWorker$start$1$1$1$1$2$1.invokeSuspend(PersistentWorker.kt:71)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)

Aug 30, 2021 4:38:18 PM worker request 0
SEVERE: Compilation failure: compile phase failed:
exception: java.io.EOFException
	at java.base/java.io.ObjectInputStream$BlockDataInputStream.readFully(ObjectInputStream.java:3317)
	at java.base/java.io.ObjectInputStream.readFully(ObjectInputStream.java:1155)
	at org.jetbrains.kotlin.utils.PluginUtilsKt.decodePluginOptions(pluginUtils.kt:42)
	at org.jetbrains.kotlin.kapt3.Kapt3CommandLineProcessor.processOption(Kapt3Plugin.kt:73)
	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParserKt.processCompilerPluginsOptions(PluginCliParser.kt:125)
	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.processPluginOptions(PluginCliParser.kt:83)
	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.loadPlugins(PluginCliParser.kt:72)
	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.loadPluginsSafe(PluginCliParser.kt:44)
	at org.jetbrains.kotlin.cli.common.CLICompiler.loadPlugins(CLICompiler.kt:183)
	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:69)
	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:52)
	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:92)
	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:44)
	at org.jetbrains.kotlin.cli.common.CLITool.exec(CLITool.kt:98)
	at io.bazel.kotlin.compiler.BazelK2JVMCompiler.exec(BazelK2JVMCompiler.kt:30)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.bazel.kotlin.builder.toolchain.KotlinToolchain$KotlinCliToolInvoker.compile(KotlinToolchain.kt:186)
	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1$1$1.invoke(compilation_task.kt:184)
	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1$1$1.invoke(compilation_task.kt:184)
	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.executeCompilerTask(CompilationTaskContext.kt:122)
	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.executeCompilerTask$default(CompilationTaskContext.kt:114)
	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1.invoke(compilation_task.kt:182)
	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1.invoke(compilation_task.kt:166)
	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.execute(CompilationTaskContext.kt:148)
	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.execute(CompilationTaskContext.kt:140)
	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt.runPlugins(compilation_task.kt:166)
	at io.bazel.kotlin.builder.tasks.jvm.KotlinJvmTaskExecutor.execute(KotlinJvmTaskExecutor.kt:54)
	at io.bazel.kotlin.builder.tasks.KotlinBuilder.executeJvmTask(KotlinBuilder.kt:232)
	at io.bazel.kotlin.builder.tasks.KotlinBuilder.build(KotlinBuilder.kt:130)
	at io.bazel.kotlin.builder.tasks.CompileKotlin.invoke(CompileKotlin.kt:27)
	at io.bazel.worker.PersistentWorker$compileWork$2$result$1.invoke(PersistentWorker.kt:96)
	at io.bazel.worker.PersistentWorker$compileWork$2$result$1.invoke(PersistentWorker.kt:94)
	at io.bazel.worker.WorkerContext$TaskContext.resultOf(WorkerContext.kt:128)
	at io.bazel.worker.WorkerContext.doTask(WorkerContext.kt:156)
	at io.bazel.worker.PersistentWorker$compileWork$2.invokeSuspend(PersistentWorker.kt:94)
	at io.bazel.worker.PersistentWorker$compileWork$2.invoke(PersistentWorker.kt)
	at io.bazel.worker.PersistentWorker$compileWork$2.invoke(PersistentWorker.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:165)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at io.bazel.worker.PersistentWorker.compileWork(PersistentWorker.kt:93)
	at io.bazel.worker.PersistentWorker.access$compileWork(PersistentWorker.kt:45)
	at io.bazel.worker.PersistentWorker$start$1$1$1$1$2$1.invokeSuspend(PersistentWorker.kt:71)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

**Comment by jeffzoch** [comment]
It seems to be related to the compiler plugins

File: src/main/starlark/release_archive/repository.bzl (hunk 1 above)
Comment (restingbull) [reply src/main/starlark/release_archive/repository.bzl:70]: My bad. Didn't think that i'd need to tell it that the change happened.

**Comment by restingbull** [comment]
> Can @Bencodes and @restingbull here do `@googlebot I consent` in a comment, to satisfy the CLAbot, please?

Need @Bencodes.

**Comment by Bencodes** [comment]
@googlebot I consent

**Comment by restingbull** [comment]
> not sure if its a bug or im doing something wrong but when i try this branch out and use language level 1.5 I see:
> 
> ```
> KotlinCompile //projects/libs/dataprotection/shared-models:shared-models-lib { kt: 1, java: 0, srcjars: 0 } for darwin failed: (Exit 1): build failed: error executing command bazel-out/host/bin/external/io_bazel_rules_kotlin/src/main/kotlin/build '--flagfile=bazel-out/darwin-fastbuild/bin/projects/libs/shared-models/shared-models-lib-kt-java.jar-0.params'
> exception: java.io.EOFException
> 	at java.base/java.io.ObjectInputStream$BlockDataInputStream.readFully(ObjectInputStream.java:3317)
> 	at java.base/java.io.ObjectInputStream.readFully(ObjectInputStream.java:1155)
> 	at org.jetbrains.kotlin.utils.PluginUtilsKt.decodePluginOptions(pluginUtils.kt:42)
> 	at org.jetbrains.kotlin.kapt3.Kapt3CommandLineProcessor.processOption(Kapt3Plugin.kt:73)
> 	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParserKt.processCompilerPluginsOptions(PluginCliParser.kt:125)
> 	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.processPluginOptions(PluginCliParser.kt:83)
> 	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.loadPlugins(PluginCliParser.kt:72)
> 	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.loadPluginsSafe(PluginCliParser.kt:44)
> 	at org.jetbrains.kotlin.cli.common.CLICompiler.loadPlugins(CLICompiler.kt:183)
> 	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:69)
> 	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:52)
> 	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:92)
> 	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:44)
> 	at org.jetbrains.kotlin.cli.common.CLITool.exec(CLITool.kt:98)
> 	at io.bazel.kotlin.compiler.BazelK2JVMCompiler.exec(BazelK2JVMCompiler.kt:30)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> 	at io.bazel.kotlin.builder.toolchain.KotlinToolchain$KotlinCliToolInvoker.compile(KotlinToolchain.kt:186)
> 	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1$1$1.invoke(compilation_task.kt:184)
> 	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1$1$1.invoke(compilation_task.kt:184)
> 	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.executeCompilerTask(CompilationTaskContext.kt:122)
> 	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.executeCompilerTask$default(CompilationTaskContext.kt:114)
> 	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1.invoke(compilation_task.kt:182)
> 	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1.invoke(compilation_task.kt:166)
> 	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.execute(CompilationTaskContext.kt:148)
> 	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.execute(CompilationTaskContext.kt:140)
> 	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt.runPlugins(compilation_task.kt:166)
> 	at io.bazel.kotlin.builder.tasks.jvm.KotlinJvmTaskExecutor.execute(KotlinJvmTaskExecutor.kt:54)
> 	at io.bazel.kotlin.builder.tasks.KotlinBuilder.executeJvmTask(KotlinBuilder.kt:232)
> 	at io.bazel.kotlin.builder.tasks.KotlinBuilder.build(KotlinBuilder.kt:130)
> 	at io.bazel.kotlin.builder.tasks.CompileKotlin.invoke(CompileKotlin.kt:27)
> 	at io.bazel.worker.PersistentWorker$compileWork$2$result$1.invoke(PersistentWorker.kt:96)
> 	at io.bazel.worker.PersistentWorker$compileWork$2$result$1.invoke(PersistentWorker.kt:94)
> 	at io.bazel.worker.WorkerContext$TaskContext.resultOf(WorkerContext.kt:128)
> 	at io.bazel.worker.WorkerContext.doTask(WorkerContext.kt:156)
> 	at io.bazel.worker.PersistentWorker$compileWork$2.invokeSuspend(PersistentWorker.kt:94)
> 	at io.bazel.worker.PersistentWorker$compileWork$2.invoke(PersistentWorker.kt)
> 	at io.bazel.worker.PersistentWorker$compileWork$2.invoke(PersistentWorker.kt)
> 	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
> 	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:165)
> 	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
> 	at io.bazel.worker.PersistentWorker.compileWork(PersistentWorker.kt:93)
> 	at io.bazel.worker.PersistentWorker.access$compileWork(PersistentWorker.kt:45)
> 	at io.bazel.worker.PersistentWorker$start$1$1$1$1$2$1.invokeSuspend(PersistentWorker.kt:71)
> 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
> 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
> 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
> 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
> 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
> 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
> 
> Aug 30, 2021 4:50:11 PM worker request 0
> SEVERE: Compilation failure: compile phase failed:
> exception: java.io.EOFException
> 	at java.base/java.io.ObjectInputStream$BlockDataInputStream.readFully(ObjectInputStream.java:3317)
> 	at java.base/java.io.ObjectInputStream.readFully(ObjectInputStream.java:1155)
> 	at org.jetbrains.kotlin.utils.PluginUtilsKt.decodePluginOptions(pluginUtils.kt:42)
> 	at org.jetbrains.kotlin.kapt3.Kapt3CommandLineProcessor.processOption(Kapt3Plugin.kt:73)
> 	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParserKt.processCompilerPluginsOptions(PluginCliParser.kt:125)
> 	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.processPluginOptions(PluginCliParser.kt:83)
> 	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.loadPlugins(PluginCliParser.kt:72)
> 	at org.jetbrains.kotlin.cli.jvm.plugins.PluginCliParser.loadPluginsSafe(PluginCliParser.kt:44)
> 	at org.jetbrains.kotlin.cli.common.CLICompiler.loadPlugins(CLICompiler.kt:183)
> 	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:69)
> 	at org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.doExecute(K2JVMCompiler.kt:52)
> 	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:92)
> 	at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:44)
> 	at org.jetbrains.kotlin.cli.common.CLITool.exec(CLITool.kt:98)
> 	at io.bazel.kotlin.compiler.BazelK2JVMCompiler.exec(BazelK2JVMCompiler.kt:30)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> 	at io.bazel.kotlin.builder.toolchain.KotlinToolchain$KotlinCliToolInvoker.compile(KotlinToolchain.kt:186)
> 	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1$1$1.invoke(compilation_task.kt:184)
> 	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1$1$1.invoke(compilation_task.kt:184)
> 	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.executeCompilerTask(CompilationTaskContext.kt:122)
> 	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.executeCompilerTask$default(CompilationTaskContext.kt:114)
> 	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1.invoke(compilation_task.kt:182)
> 	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt$runPlugins$1.invoke(compilation_task.kt:166)
> 	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.execute(CompilationTaskContext.kt:148)
> 	at io.bazel.kotlin.builder.toolchain.CompilationTaskContext.execute(CompilationTaskContext.kt:140)
> 	at io.bazel.kotlin.builder.tasks.jvm.Compilation_taskKt.runPlugins(compilation_task.kt:166)
> 	at io.bazel.kotlin.builder.tasks.jvm.KotlinJvmTaskExecutor.execute(KotlinJvmTaskExecutor.kt:54)
> 	at io.bazel.kotlin.builder.tasks.KotlinBuilder.executeJvmTask(KotlinBuilder.kt:232)
> 	at io.bazel.kotlin.builder.tasks.KotlinBuilder.build(KotlinBuilder.kt:130)
> 	at io.bazel.kotlin.builder.tasks.CompileKotlin.invoke(CompileKotlin.kt:27)
> 	at io.bazel.worker.PersistentWorker$compileWork$2$result$1.invoke(PersistentWorker.kt:96)
> 	at io.bazel.worker.PersistentWorker$compileWork$2$result$1.invoke(PersistentWorker.kt:94)
> 	at io.bazel.worker.WorkerContext$TaskContext.resultOf(WorkerContext.kt:128)
> 	at io.bazel.worker.WorkerContext.doTask(WorkerContext.kt:156)
> 	at io.bazel.worker.PersistentWorker$compileWork$2.invokeSuspend(PersistentWorker.kt:94)
> 	at io.bazel.worker.PersistentWorker$compileWork$2.invoke(PersistentWorker.kt)
> 	at io.bazel.worker.PersistentWorker$compileWork$2.invoke(PersistentWorker.kt)
> 	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
> 	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:165)
> 	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
> 	at io.bazel.worker.PersistentWorker.compileWork(PersistentWorker.kt:93)
> 	at io.bazel.worker.PersistentWorker.access$compileWork(PersistentWorker.kt:45)
> 	at io.bazel.worker.PersistentWorker$start$1$1$1$1$2$1.invokeSuspend(PersistentWorker.kt:71)
> 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
> 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
> 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
> 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
> 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
> 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
> ```
> 
> and the file in question:
> 
> ```
> --target_label
> //projects/libs/dataprotection/shared-models:shared-models-lib
> --rule_kind
> kt_jvm_library
> --kotlin_module_name
> projects_libs_dataprotection_shared-models-shared-models-lib
> --kotlin_jvm_target
> 11
> --kotlin_api_version
> 1.5
> --kotlin_language_version
> 1.5
> --kotlin_passthrough_flags
> -Xuse-experimental=kotlin.Experimental
> --kotlin_debug_tags
> --output
> bazel-out/darwin-fastbuild/bin/projects/libs/dataprotection/shared-models/shared-models-lib-kt-java.jar
> --kotlin_output_jdeps
> bazel-out/darwin-fastbuild/bin/projects/libs/dataprotection/shared-models/shared-models-lib-kt.jdeps
> --output_deps_proto
> bazel-out/darwin-fastbuild/bin/projects/libs/dataprotection/shared-models/shared-models-lib-java.jdeps
> --generated_java_srcjar
> bazel-out/darwin-fastbuild/bin/projects/libs/dataprotection/shared-models/shared-models-lib-gensrc.jar
> --kotlin_passthrough_flags
> -Xinline-classes
> -Xallow-result-return-type
> -Xjvm-default=enable
> --direct_dependencies
> external/com_github_jetbrains_kotlin/lib/annotations-13.0.jar
> external/com_github_jetbrains_kotlin/lib/kotlin-stdlib.jar
> external/com_github_jetbrains_kotlin/lib/kotlin-stdlib-jdk7.jar
> external/com_github_jetbrains_kotlin/lib/kotlin-stdlib-jdk8.jar
> --strict_kotlin_deps
> off
> --classpath
> external/com_github_jetbrains_kotlin/lib/annotations-13.0.jar
> external/com_github_jetbrains_kotlin/lib/kotlin-stdlib.jar
> external/com_github_jetbrains_kotlin/lib/kotlin-stdlib-jdk7.jar
> external/com_github_jetbrains_kotlin/lib/kotlin-stdlib-jdk8.jar
> --reduced_classpath_mode
> NONE
> --sources
> projects/libs/dataprotection/shared-models/src/main/java/com/salesforce/dataprotection/ticketdetail/TicketDetail.kt
> --instrument_coverage
> false
> --stubs_plugin
> external/com_github_jetbrains_kotlin/lib/allopen-compiler-plugin.jar
> external/com_github_jetbrains_kotlin/lib/noarg-compiler-plugin.jar
> --stubs_plugin_classpath
> external/com_github_jetbrains_kotlin/lib/allopen-compiler-plugin.jar
> external/com_github_jetbrains_kotlin/lib/noarg-compiler-plugin.jar
> --stubs_plugin_options
> org.jetbrains.kotlin.allopen:preset=spring
> org.jetbrains.kotlin.noarg:preset=jpa
> --compiler_plugin
> external/com_github_jetbrains_kotlin/lib/allopen-compiler-plugin.jar
> external/com_github_jetbrains_kotlin/lib/noarg-compiler-plugin.jar
> --compiler_plugin_classpath
> external/com_github_jetbrains_kotlin/lib/allopen-compiler-plugin.jar
> external/com_github_jetbrains_kotlin/lib/noarg-compiler-plugin.jar
> --compiler_plugin_options
> org.jetbrains.kotlin.allopen:preset=spring
> org.jetbrains.kotlin.noarg:preset=jpa
> --build_java
> true
> --build_kotlin
> true
> ```

Hm. The format may have changed. I'll take a look -- should bump the examples to use 1.5

**Pull request merged**